themes_cern/indico_themes_cern/themes/ngt.scss

```diff
@@ -0,0 +1,13 @@
+@use 'base/palette' as *;
+
+$header-bg-color: #15266c;
+
+@import 'themes/indico';
+
+div.event-header {
+  background: url('themes_cern:ngt_logo.png'), url('themes_cern:ngt_background.png');
```

Comment:
TIL you can have multiple background images

Reply:
Thank @tomasr8 for that ;)